kolibri/plugins/learn/assets/src/views/SuggestedTime.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+
+  <div class="time-context">
+    <div v-if="time > 0">
+      {{ $tr('shortTime', { time: 120 }) }}
+    </div>
+    <div v-if="time > 0">
+      {{ $tr('suggestedTimeLabel', { time: 120 }) }}
+    </div>
+    {{ $tr('suggestedTimeTooltip') }}
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'SuggestedTime',
+    props: {
+      /* Time in ms */
+      time: {
+        type: Number,
+        default: 0,
+      },
+    },
+    $trs: {
+      suggestedTimeTooltip: {
+        message: 'Suggested time to complete',
+        context:
+          'A tooltip explaning what the clock icon and time indicates about the resource  they are viewing',
+      },
+      shortTime: {
+        message: '{time} {time, plural, one {minute} other {minutes}}',
```

Comment:
I noticed that and figured that when I come back to it (post strings-getting-in rush) then I'll conditionalize showing any strings at all on whether or not we have > 0 passed in

Reply:
Ah - I already had it conditionalized.

@radinamatic - I think that TimeDuration will work the best but it isn't accurate - it rounds down. If @jtamiace and @rtibbles are okay with the way it rounds, then this component can be the "Suggested time to complete:" followed by the TimeDuration component.

Reply:
Understood, thank you @nucleogenesis! 
That type of time rounding does not seem like a big concern to me, but will defer to @jtamiace & @rtibbles.

Reply:
I think if it's an issue we can probably further conditionalize TimeDuration to round up rather than down?

Reply:
It seems okay for now since it still gives a ballpark about expected length, but I'd hope accuracy is something we can improve in the future. 

Any issue I can foresee is around planning and scheduling around learning with limited time and the resource taking longer than they expected, but I can't imagine that causing too many issues because it's just an estimate and people will engage resources at their own discernment.

Reply:
> further conditionalize TimeDuration to round up rather than down

And I totally misread that, and thought that 1:45h was rounded to 2h... 🤦🏽‍♀️ 
Yes, rounding up would be more accurate in this case!

Reply:
> I'd hope accuracy is something we can improve in the future.

We can also conditionalize the TimeDuration component's threshold for when it starts displaying 'hours', so we could make it display whole minutes for up to 120 minutes, and only then start displaying hours. Once we get past 120 minutes, hopefully any time durations are very ballpark anyway.

Reply:
I just pushed it up with only the "Suggested time to complete" string.

In follow up I can use TimeDuration to get things looking how we want - but at this point this should be a go for strings I think. Anybody feel free to force push anything you need to get this merged!

Reply:
Thanks for the update - and apologies for brigading your tiny PR!